index.html

```diff
@@ -745,10 +745,9 @@ <h3>Credentials</h3>
 	       credentialSubject of Pat, who has an alumniOf property
 	       with value of Example University.  The Proof Graph has
 	       Signature 456 with 5 properties: 'type' of
-	       RsaSignature2018, 'creator' of Example University
-	       Public Key 7, 'created' of 2017-06-18T21:19:10Z,
-	       'nonce' of 348dj239dsj328, and 'signatureValue' of
-	       BavEll0...3JT24=">
+	       RsaSignature2018, 'verificationMethod' Example University
+	       Public Key 7, 'created' of 2017-06-18T21:19:10Z, and 'jws'
+         of BavEll0...3JT24=">
```

Comment:
Missing opening quote?

Reply:
Well, no, that final quote is for the HTML attribute on that tag ... it was unchanged from before, but now I've added single quotes around the 'jws' value so it's less confusing (hopefully).